background-concealment-demo-py/BackgroundConcealmentDemo.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            return cv.GMatDesc(desc.depth, desc.dims[1], (desc.dims[3], desc.dims[2]))
</pre><p>As it is kindof tutorial, it can be useful to show what the input arguments are here in a GMatDesc constructor.</p><p>And thankfully python has this feature, I mean, to write</p><pre class="ql-syntax" spellcheck="false">return cv.GMatDesc(depth=desc.depth, channels=desc.dims[1], size=(desc.dims[3], desc.dims[2]))
# I'm not sure what their names are, just to show the point
</pre><p>This increases readability, I think</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TolyaTalamanov/g-api-collection/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>